src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -72,15 +97,23 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      *                  if `someDate` is today;
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate currentDate = LocalDate.now();
+        if (currentDate.isAfter(someDate)) {
+            return someDate + " is before " + currentDate;
+        } else if (currentDate.isBefore(someDate)) {
```

Comment:
do we need `else` after `return` in `if` block?

Reply:
Nope) Fixed